.codedeploy/scripts/add_node_specific_config

```diff
@@ -23,10 +23,10 @@ FLAGS=
 
 case "$NAME" in
 	vln)
-		FLAGS="--ws-port 80 --rpc-port 8080 --rpc-external --ws-external --rpc-cors all --node-key $VLN1_KEY"
+		FLAGS="--ws-port 80 --rpc-port 8080 --rpc-external --ws-external --rpc-cors all"
```

Comment:
still missing the `--node-key` part, without it the peerId is random and collators can't find the bootnode